horace_core/admin/cache.py

```diff
@@ -116,9 +113,7 @@ def process_file(argi):
     buf_size = argi['buffer']
 
     # Estimate the file size
```

Comment:
Python has tools for getting the file size. Why this?

Reply:
probably Python 2 did not have it.

Reply:
No, [it has](https://docs.python.org/3/library/os.html#os.stat) been in there since at least 2.2 which is where the docs history stops

Reply:
modified.